src/official-widgets/merchandise-search-bar/README.md

```diff
@@ -0,0 +1,21 @@
+# Merchandise Search Bar widget
+
+![status](https://img.shields.io/badge/status-development-yellow)
+![stable](https://img.shields.io/badge/since-1.0.5-blue)
```

Comment:
This is almost certainly not yet stable in 1.0.6 (and definitely not 1.0.5!), so this badge should be removed